### PR TITLE
Add live migrations to index page

### PR DESCRIPTION
**Pull request by VineethReddy02**
# Description

Include live migrations in index/parent page. 

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: migrate/index.md [hunk 1]
```diff
@@ -55,3 +70,4 @@ If you encounter any difficulties while migrating, consult the
 [pg-dump-restore]: /migrate/:currentVersion:/pg-dump-and-restore/
 [parallel-copy]: /use-timescale/:currentVersion:/ingest-data/import-csv/
 [troubleshooting]: /migrate/:currentVersion:/troubleshooting/
+[live-migrations]: /migrate/:currentVersion:/live-migrations/
```
Comment (VineethReddy02) [inline migrate/index.md:73]: Note to self, before merging verifying this path is aligned with @alejandrodnm PR.

File: migrate/index.md [hunk 2]
```diff
@@ -18,35 +18,50 @@ which are:
...
-pg_dump and pg_restore][pg-dump-restore].
-
-If you are looking for a low-downtime alternative (downtime on the order of
-minutes), are willing to modify your ingestion code, and the bulk of your data
-is stored in time-series tables, you can use the [dual-write and backfill][dual-write]
-strategy for a low-downtime migration. This strategy also works if you are not
-migrating from PostgreSQL.
+`pg_restore` commands. 
+
+You can also use these tools to migrate your data from Managed Service for TimescaleDB, from a self-hosted TimescaleDB instance, or from another
```
Comment (arajkumar) [inline migrate/index.md:34]: We don't support Timescale as a source yet.

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: migrate/index.md (hunk 2 above)
Comment (VineethReddy02) [reply migrate/index.md:34]: Yes, thats the reason I have added table in the top mentioning the supported sources. In this sentence the context is we have tools that support migrations from Postgres, Timescale and others.

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**